content/en/security_platform/cspm/findings.md

```diff
@@ -44,11 +44,33 @@ A finding is the primary primitive for a rule evaluation against a resource. Eve
 
 {{< img src="security_platform/cspm/findings/posture-management-overview.png" alt="An overview of the Posture Management Findings page" style="width:100%;">}}
 
-## Triage, correlate, and alert on misconfigurations with Signals
+### Signals
+
+Signals are security alerts that Datadog generates and displays in the [Signals Explorer][4]. CSPM signals trigger when Datadog generates an `evaluation:fail` finding for a cloud or infrastructure configuration rule.
+
+In order to consume findings in logical groupings and mitigate the potential for alert fatigue, you have full flexibility to change how signals are triggered for each individual resource every time a resource fails a rule in a new cloud account or each time a resource is misconfigured in a service. Regardless of which grouping logic you choose for signal generation, opening a signal always displays the up-to-date list of findings that are failing for this rule.
 
 Clicking on an individual finding that has **failed** to see details about the misconfigured resource, the rule description, its framework or industry benchmark mapping, and suggested remediation steps.
 
-{{< img src="security_platform/cspm/findings/signal-overview.png" alt="Failed signals in the side panel" style="width:65%;">}}
+{{< img src="security_platform/cspm/findings/signal.png" alt="An example of a Signal" style="width:80%;">}}
+
+From the top of the side panel, you can configure a rule, or send a notification to your colleagues by email, Slack, Teams, Pagerduty, Servicenow, Jira, and webhooks.
+
+The top portion of the finding side panel also displays key information about where the misconfiguration(s) are occurring: an individual resource, a service, or a cloud account. See all tags associated with this grouping so you can quickly find this area within your infrastructure for deeper inspection. It also highlights the relevant compliance controls associated with the rule, and when the signal was first triggered by a finding.
```

Comment:
bold it since we're referring to a specific UI text object

```suggestion
The top portion of the **Findings** side panel also displays key information about where misconfigurations are occurring: an individual resource, a service, or a cloud account. See all tags associated with this grouping so you can quickly find this area within your infrastructure for deeper inspection. It also highlights the relevant compliance controls associated with the rule, and when the signal was first triggered by a finding.
```